vendor/psmodules/Cmder.ps1

```diff
@@ -1,18 +1,110 @@
-function Configure-Git($GIT_INSTALL_ROOT){
-  $env:Path += $(";" + $GIT_INSTALL_ROOT + "\cmd")
+function readVersion($gitPath) {
+    $gitExecutable = "${gitPath}\git.exe"
 
-  # Add "$GIT_INSTALL_ROOT\usr\bin" to the path if exists and not done already
-  $GIT_INSTALL_ROOT_ESC=$GIT_INSTALL_ROOT.replace('\','\\')
-  if ((test-path "$GIT_INSTALL_ROOT\usr\bin") -and -not ($env:path -match "$GIT_INSTALL_ROOT_ESC\\usr\\bin")) {
-      $env:path = "$env:path;$GIT_INSTALL_ROOT\usr\bin"
-  }
+    if (!(test-path "$gitExecutable")) {
+        return $null
+    }
+
+    $gitVersion = (cmd /c "${gitExecutable}" --version)
+
+    if ($gitVersion -match 'git version') {
+        ($trash1, $trash2, $gitVersion) = $gitVersion.split(' ', 3)
+    } else {
+        pause
```

Comment:
With all the other `write-host` calls commented out `pause` here isn't going to have any context for the user.

Reply:
I have no idea why that is there probably just an oversight while debugging.

Reply:
fixed